apa-cv.csl

```diff
@@ -1,15 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
   <info>
-    <title>American Psychological Association 6th edition (curriculum vitae, sorted by descending date)</title>
+    <title>American Psychological Association 6th edition (curriculum vitae, citation becomes reference, no bibliography)</title>
     <title-short>APA</title-short>
     <id>http://www.zotero.org/styles/apa-cv</id>
     <link href="http://www.zotero.org/styles/apa-cv" rel="self"/>
     <link href="http://owl.english.purdue.edu/owl/resource/560/01/" rel="documentation"/>
-    <author>
+    <autho>
```

Comment:
@reagle, `<autho>` should read `<author>`. I always recommend running the style through https://validator.citationstyles.org/ after hand-editing to find typos like this.